edexOsgi/com.raytheon.uf.edex.plugin.loctables/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
```

Comment:
Is this file totally necessary as well?  Becuase It also seems to point to java 1.8